packages/x-data-grid/src/components/index.ts

```diff
@@ -18,3 +18,4 @@ export { GridPagination } from './GridPagination';
 export * from './GridRowCount';
 export * from './GridRow';
 export * from './GridSelectedRowCount';
+export * from './GridSkeletonLoadingOverlay';
```

Comment:
Any thoughts on whether we should export this or not?

Reply:
When in doubt, I would avoid adding stuff to the public API, because it's unchangeable once published.

Reply:
Have undone this for now. If anyone feels strongly about exporting it let me know.